Multiplex.cpp

```diff
@@ -53,23 +55,61 @@ size_t Multiplex::write(uint8_t c)
   return n;
 }
 
+size_t Multiplex::write(const uint8_t *buffer, size_t size) {
+  uint8_t n = 0;
+  for (uint8_t i = 0; i < _count; i++)
+  {
+    if (_enabled[i])
+    {
+      n += _stream[i]->write(buffer, size);
+    }
+  }
+  return n;
+}
+
+// private
+uint8_t Multiplex::index(Print *stream) 
+{
+  uint8_t i = 0;
+  for (; i < _count; i++)
+  {
+    if (stream == _stream[i]) 
+    {
+      return i;
+    }
+  }
+  return 0xFF;
+}
 
 void Multiplex::enable(uint8_t n)
 {
-  if (n < _count) _enabled[n] = true;
+  if (n != 0xFF && n < _count) _enabled[n] = true;
```

Comment:
reversing the condition is faster, no need to fix as this is minor than minor
if ( n < _count  && n != 0xFF) _enabled[n] = true;